lib/nostrum/struct/message.ex

```diff
@@ -61,6 +67,11 @@ defmodule Nostrum.Struct.Message do
   @typedoc "Whether this messsage mentions everyone"
   @type mention_everyone :: boolean
 
+  @typedoc """
+  List of Channel mention objects.
+  """
```

Comment:
This docstring should match the format of the other docstrings, specifically, the lack of trailing period. For example:

```elixir
  @typedoc "List of channel mention objects"
```

or, more descriptive:

```elixir
  @typedoc "Channels mentioned in this message"
```